modules/apps/batch-planner/batch-planner-test/src/testIntegration/java/com/liferay/batch/planner/batch/engine/broker/test/BatchEngineBrokerTest.java

```diff
@@ -370,11 +373,79 @@ public void testExportObjectDefinition() throws Exception {
 			actualJsonNode);
 	}
 
+	@Test
```

Comment:
Maybe it is the time (not sure if this PR or a following-up PR) to include tests for the `SiteScopeResourceImpl`